### PR TITLE
Adding missing multiapi tag

### LINKED ISSUE #1023: Client avoidance of duplicate names produces duplicate files
After this PR : https://github.com/Azure/autorest.go/commit/3458098afd88db38c357921cbd964aa72685a61d  

there is an unexpected behavior in the generation for example: ` azure-rest-api-specs\specification\automation `

It results in creating duplicate files with the same content the only difference is that one has the extra `Group` suffix.  ( **runbookdraft.go**, **runbookdraftgroup.go** )
 
As a side effect the build failures because teh future names are getting updated to include the `Group` part although they are referenced in the **runbookdraft** files .

```
.\runbookdraft.go:49:158: undefined: RunbookDraftCreateOrUpdateFuture
.\runbookdraft.go:96:82: undefined: RunbookDraftCreateOrUpdateFuture
.\runbookdraft.go:270:121: undefined: RunbookDraftPublishFuture
.\runbookdraft.go:316:75: undefined: RunbookDraftPublishFuture
```

### PR DESCRIPTION
**Pull request by marstr**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

Fixes Azure/azure-sdk-for-go#1023

---

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/automation/resource-manager/readme.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ go:
 
 ### Go multi-api
 
-``` yaml $(go)
+``` yaml $(go) && $(multiapi)
```
Comment (vladbarosan) [inline specification/automation/resource-manager/readme.md]: Not sure how to review this, no idea where I can see what the `multiapi` tag does :)

Comment (marstr) [reply specification/automation/resource-manager/readme.md]: The multi-api tag should prevent `batch` from being invoked unless it was called for. Azure/azure-sdk-for-go#1023 is cause by multiple API Versions being generated when only a single one is called for.

Comment (jhendrixMSFT) [reply specification/automation/resource-manager/readme.md]: Is this the only config file that needs to be fixed?  I see batch all over the place.

Comment (marstr) [reply specification/automation/resource-manager/readme.md]: I did a search for all instances of `$(go)`, and didn't find any other instances where `batch` was unprotected by an accompanying `$(multiapi)`.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**